shotgun_api3/shotgun.py

```diff
@@ -1519,7 +1534,7 @@ def get_attachment_download_url(self, attachment):
                 None, None, None))
         return url
 
-    def authenticate_human_user(self, user_login, user_password):
+    def authenticate_human_user(self, user_login, user_password, auth_token):
```

Comment:
Requiring an auth_token here is a non-backward compatible change that could break client scripts.  It should have a default value so existing scripts can be used without change.